.env.example

```
@@ -9,6 +9,8 @@ DEBUG=True
 # For Postgres: postgresql://user:password@host:port/dbname
 DATABASE_URL=sqlite:///db.sqlite3
 
-# Email Configuration for OTP (Optional)
-EMAIL_HOST_USER=
-EMAIL_HOST_PASSWORD=
+# Email Configuration for OTP and Password Reset Emails
+# Use Gmail App Passwords (not normal Gmail password)
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)